CMakeLists.txt

```diff
@@ -49,6 +49,9 @@ endif()
 #######################################################################
 
 set(CMAKE_CXX_STANDARD 17)
+if(MSVC)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Zc:__cplusplus")
```

Comment:
This is _required_ for the current version of libdjinterop (which looks only at `__cplusplus` and doesn't use any `_MSVC_LANG` hacks), but may be considered a good idea to enable anyway.

FWIW, if used on an older version of MSVC, this prints an "unrecognised option" message and carries on anyway.

Reply:
Shouldn't it suffice to add this to the libdjinterop CMakeLists.txt? Or is this required for libdjinterop includes? If so, will building Mixxx still work if an older MSVC version is used that doesn't support the flag?

Reply:
This flag is already set in libdjinterop's own CMakeLists.txt file, but it is indeed also required to use libdjinterop's public includes, which is why it's required in Mixxx in order to use libdjinterop.

The `/Zc:__cplusplus` flag is supported from VS 2017 onwards, which appears to be the minimum-supported version to build Mixxx (according to the [wiki](https://github.com/mixxxdj/mixxx/wiki/Compiling%20on%20Windows)).  That was my reasoning for adding the flag for the whole MSVC build - hope this is ok.